src/main/java/at/hannibal2/skyhanni/features/misc/update/GuiOptionEditorUpdateCheck.kt

```diff
@@ -31,25 +34,28 @@ class GuiOptionEditorUpdateCheck(option: ProcessedOption) : GuiOptionEditor(opti
         }
         button.render(getButtonPosition(width), 10)
 
+        if (UpdateManager.updateState != UpdateManager.UpdateState.NONE) {
+            changelog.render(getChangelogPosition(width), 30)
+        }
+
+        val widthRemaining = width - button.width - 10
+
         if (UpdateManager.updateState == UpdateManager.UpdateState.DOWNLOADED) {
-            TextRenderUtils.drawStringCentered(
+            TextRenderUtils.drawStringCenteredScaledMaxWidth(
                 "${GREEN}The update will be installed after your next restart.",
                 fr,
-                width / 2F,
+                widthRemaining / 2F,
                 40F,
                 true,
+                widthRemaining,
                 -1
             )
         }
 
-        val widthRemaining = width - button.width - 10
-
         GlStateManager.scale(2F, 2F, 1F)
-        val currentVersion = SkyHanniMod.version
         val sameVersion = currentVersion.equals(nextVersion, true)
         TextRenderUtils.drawStringCenteredScaledMaxWidth(
-            "${if (UpdateManager.updateState == UpdateManager.UpdateState.NONE) GREEN else RED}$currentVersion" +
-                if (nextVersion != null && !sameVersion) "➜ ${GREEN}${nextVersion}" else "",
+            "${if (UpdateManager.updateState == UpdateManager.UpdateState.NONE) GREEN else RED}$currentVersion" + if (nextVersion != null && !sameVersion) "➜ ${GREEN}${nextVersion}" else "",
```

Comment:
please use the color codes and fix the existing things in this class from using EnumChatFormatting as to keep consistent with almost every other chatformatting usages in the codebase.

Reply:
I didn't change anything here, the formatting just removed the wrap and to put it simply the rendering of that class uses only NEU calls. So the would need to be reworked as a whole.

Reply:
I know you didnt change anything here but I am saying while you're here fix it, because this is the only class in the whole codebase that actually uses the EnumChatFormatting inline in a chat unlike everywhere else it uses the section symbols and the color codes